### PR TITLE
chore(repo): update nx to 20.2.0-beta.3

### PR DESCRIPTION
**Pull request by FrozenPandaz**
## Current Behavior

This repo uses Nx `20.2.0-beta.2`.

## Expected Behavior

This repo uses Nx `20.2.0-beta.3`.

## Related Issue(s)

Fixes #

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,9 +55,9 @@
     "@jest/types": "^29.4.1",
     "@module-federation/enhanced": "0.7.6",
     "@module-federation/sdk": "0.7.6",
-    "@monodon/rust": "2.0.0-beta.1",
-    "@napi-rs/cli": "3.0.0-alpha.56",
-    "@napi-rs/wasm-runtime": "0.2.4",
+    "@monodon/rust": "2.1.1",
+    "@napi-rs/cli": "3.0.0-alpha.63",
```
Comment (Cammisuli) [inline package.json:59]: Revert this change, as I think it's not a good release from the napi side. It doesn't output wasm.js files

**Review by Cammisuli — CHANGES_REQUESTED** [review]

**Review by AgentEnder — APPROVED** [review]

**Review by AgentEnder — APPROVED** [review]
